solution/1200-1299/1251.Average Selling Price/README.md

```diff
@@ -83,7 +83,14 @@ Result table:
 ### **SQL**
 
 ```sql
-
+SELECT p.product_id,
+       Round(( Sum(u.units * p.price) + 0.0 ) / ( Sum(units) + 0.0 ), 2)
+       avg_price
```

Comment:
```suggestion
       average_price
```

Reply:
Write an SQL query to find the average selling price for each product. `average_price` should be rounded to 2 decimal places.

Reply:
Done ✅